README.md

```diff
@@ -120,14 +111,14 @@ options = {"grid": grid}
 # Pass the new grid to the environment (for the next game)
 env.reset(options=options)
 ```
-Grids are encoded using these symbols:
-- `.` for cells where you can move your army
-- `#` for mountains (terrain that can not be passed)
-- `A,B` are positions of generals
-- digits `0-9` represent cities with the cost calculated as `(40 + digit)`
-
-## 🔬 Replays
-We can store replays and then analyze them. `Replay` class handles replay related functionality.
+Grids are created using a string format where:
+- `.` represents passable terrain
+- `#` indicates impassable mountains
+- `A, B` mark the positions of generals
+- digits `0-9` represent cities, where the number is a city `cost` calculated as `40 + digit`
```

Comment:
Something like "the number specifies amount of neutral army in the city, which is calculated as ..." sound more precise.